client/accounting.go

```diff
@@ -44,13 +44,18 @@ func (x ResBalanceGet) Amount() accounting.Decimal {
 // Context is required and must not be nil. It is used for network communication.
 //
 // Return errors:
-//   - [ErrMissingAccount]
+//   - [ErrMissingAccount];
+//   - [ErrMissingSigner].
 func (c *Client) BalanceGet(ctx context.Context, prm PrmBalanceGet) (*ResBalanceGet, error) {
 	switch {
 	case !prm.accountSet:
 		return nil, ErrMissingAccount
 	}
 
+	if c.prm.signer == nil {
+		return nil, ErrMissingSigner
```

Comment:
How did it work before this patch?

Reply:
Actually, not a question, I see `initCallContext`. Maybe we could provide ephemeral key for these simple getters in this case, but that's a different story.